text/0091-logs-vocabulary.md

```diff
@@ -0,0 +1,49 @@
+# Logs: Vocabulary
+
+This documents defines the vocabulary for logs to be used across OpenTelemetry project.
+
+## Motivation
+
+We need a common language and common understanding of terms that we use to
+avoid the chaos experienced by the builders of the Tower of Babel.
+
+## Proposal
+
+OpenTelemetry specification already contains a [vocabulary](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md)
+for Traces, Metrics and other relevant concepts.
+
+This proposal is to add the following concepts to the vocabulary.
+
+### Log Record
+
+A recording of an event. Typically the record includes a timestamp indicating
+when the event happened as well as other data that describes what happened,
+where it happened, etc.
+
+### Embedded Log 
+
+A log record embedded inside a [Span](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#span) 
+object, in the [Events](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#add-events) list.
+
+### Standalone Log
+
+A log record that is not embedded inside a Span and is recorded elsewhere.
+
+### Log Attributes
+
+Key/value pairs contained in a Log Record.
+
+### Structured Logs
+
+Logs that are recorded in a format that has a well-defined structure that allows
+to differentiate between different elements of a Log Record (e.g. the Timestamp,
+the Attributes, etc). For example [Syslog, RFC5425](https://tools.ietf.org/html/rfc5424)
+protocol defines `structured-data` format.
```

Comment:
```suggestion
the Attributes, etc). The _Syslog protocol_ ([RFC 5425](https://tools.ietf.org/html/rfc5424)),
for example, defines a `structured-data` format.
```

the rendered version looks a bit odd otherwise

Reply:
Fixed.